mega-linter-runner/lib/options.js

```diff
@@ -137,7 +137,12 @@ export const optionsDefinition = optionator.default({
     {
       option: "remove-container",
       type: "Boolean",
-      description: "Remove MegaLinter Docker container when done",
+      description: "Remove MegaLinter Docker container when done (default: true since v7.8.0)",
+    },
+    {
+      option: "no-remove-container",
```

Comment:
@nvuillam should this new option be set to be mutually exclusive like other sets below?

And shouldn’t release be mutually exclusive with image (you can’t add a tag to an already full name)?

Reply:
@echoix indeed you are right :)